### DISCUSSION TITLE
Duration for videos

### DISCUSSION THREAD

**Discussion by HikaruIchijyo** [robertodoering/harpy#742] [Ideas]
Duration of the current video before opening it like how it is shown on twitter?

**Comment by robertodoering** [comment]
This definitely makes a lot of sense, thanks for the suggestion.
I just checked and we do get the duration before loading the video so it's easy to implement. I've created the issue [#744](https://github.com/robertodoering/harpy/issues/744) and added it to the board

**Reply by HikaruIchijyo** [reply]
np

**Comment by HikaruIchijyo** [comment]
> This definitely makes a lot of sense, thanks for the suggestion.
> I just checked and we do get the duration before loading the video so it's easy to implement. I've created the issue [#744](https://github.com/robertodoering/harpy/issues/744) and added it to the board

do u have todo list sorted by priority?
if not y not make one? 
so that we can know what to expect in the next version or so

**Reply by dnestico** [reply]
He does https://github.com/robertodoering/harpy/projects/1

**Reply by robertodoering** [reply]
The 'next' column is kinda sorted by priority, but it depends. When I don't have a lot of time to work on features (like recently) I'll more likely work on low hanging fruits (like this one).
